src/utils/useBroadcast.js

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react';
+
+export default function useBroadcast(broadcast, defaultValue) {
+  const [value, setValue] = useState(broadcast ? broadcast.getState() : defaultValue);
```

Comment:
this will just throw if hooks aren't available; should it use https://github.com/airbnb/react-with-styles/pull/224/files#diff-56c9bb27cfc316ce0cfd3db74a78d2e0R18-R20?